Added requirements and new options for Next 15 (#1257)
edgio-docs/edgio-docs: guides/applications/v7/sites_frameworks/getting_started/next.md

| `optimizeServerBuild`        | Optimizes the server build by bundling all server assets and decreasing the overall startup time. This option has no effect on apps with the {{ PRODUCT }} cloud build. This option is set to `false` for Next 13.x apps.              | `true`  |
| `proxyToServerlessByDefault` | Reduces the number of generated rules by adding the default catch-all rule that proxies all requests to Next.js to the {{ PRODUCT }} cloud. Set this option to `false` if you want to proxy all unmatched pages to a different origin. | `true`  |
| `enforceTrailingSlash`       | Adds rules with Next's internal redirects that either add or remove a trailing slash. When set to `false`, the redirect is performed only by the Next.js server itself and doesn't affect other origins.                               | `true`  |

> | `turbopack`                  | Enables TurboPack bundler instead of default Webpack for your Next.js app in the development mode. This option requires Next.js 13 and newer.                                                                                          | `false` |

Reviewer: ```suggestion
| `turbopack`                  | Enables the TurboPack bundler instead of the default Webpack when running your Next.js app in development mode. This option requires Next.js 13 or later.                                                                                          | `false` |
```
